lib/bolt_server/transport_app.rb

```diff
@@ -375,6 +375,27 @@ def make_winrm_target(target_hash)
       end
     end
 
+    # Fetches the list of tasks for an environment
+    #
+    # @param environment [String] the environment to fetch the list of tasks from
+    get '/tasks' do
+      in_pe_pal_env(params['environment']) do |pal|
+        tasks = pal.list_tasks
+        tasks_response = tasks.map { |task_name, _description|
+          {
+            'name' => task_name,
+            'environment' => [{ 'name' => params['environment'] }]
```

Comment:
This just seems like it belongs in clojure code. I dont think addition of environment is necessary for bolt-server to send over the wire.